### ISSUE TITLE
Darwin core files when treatment capture fails

### ISSUE THREAD

**Issue by flsimoes** [gsautter/goldengate-imagine#906]
In the event of getting a batch processed file that does not capture any treatment, and as a consequence doesn't create a DarwinCore file, what happens when we manually correct the IMF to include a treatment? Should we `Export > Export DarwinCore Archive` manually then?

Here's an example where I manually corrected the file by adding the treatments: Issue 0734 [IMF](https://drive.google.com/open?id=1o0wAvlz66Jtax4ykTkgDfRZoIUXHJ0AU), [PDF](https://drive.google.com/open?id=1E6BGDXI2XBLvfEnjkIM64qVGRDTGC-U9), [FT.ZIP](https://drive.google.com/open?id=1i2oiuBThCZoE6IXhqUH8aKhHrugN7Max)

**Comment by myrmoteras** [comment]
if there is no treatment, then there is none. You just upload the file as usual. The DWCA is not needed.
if there is a treatment that has not been covered, you mark it up, finish the processing and upload to TB as usual. The DWCA is not needed.

**Comment by flsimoes** [comment]
In the latter case, we would be missing a DarwinCore file (as I did with the example I linked above). That's why I'm asking whether I should create this file by exporting the IMF to a DC.

This case is quite different from when the template captures something like 9 out of 11 treatments, upon which it does create a DC file. After that I know I just need to add the remaining treatments, save the IMF and upload it to TB.
